src/Service/Google.php

```diff
@@ -44,13 +56,16 @@ public function getExchangeRate(ExchangeRateQuery $exchangeQuery)
         }
 
         $xpath = new \DOMXPath($document);
-        $nodes = $xpath->query('//span[@class="bld"]');
+        $nodes = $xpath->query('//div[@class="vk_ans vk_bk"]');
 
-        if (0 === $nodes->length) {
+        if (1 !== $nodes->length) {
             throw new Exception('The currency is not supported or Google changed the response format');
         }
 
         $nodeContent = $nodes->item(0)->textContent;
+
+        // Beware of "3 417.36111 Colombian pesos", with a non breaking space
+        $nodeContent = strtr($nodeContent,["\xc2\xa0" => '']);
```

Comment:
Non-breaking space. They use it instead of an common space to separate the thousands.